add links to image analysis resources

haesleinhuepf opened the pull request:
Hi @bethac07 et al,

I just discovered your amazing resource [via twitter](https://twitter.com/BoSoxBioBeth/status/1669023251631333376?s=20). Hence, I'm just proposing to add two links to the image analysis resource list.

Let me know what you think!

Best,
Robert

bethac07 commented:
Glad you like it! Please do share as you like, and we're always very happy for suggested improvements! 

The first link is great; the second link is also great but maybe better suited to the [napari section](https://github.com/COBA-NIH/MicroscopyForBeginnersReferenceGuide/blob/main/03_Image_analysis/GeneralUseSoftware.md#-napari), if that's not too much of a pain to ask? We've generally tried to keep tool-specific stuff to the tool sections.

Also, feel free to add your name to our [contributors section](https://github.com/COBA-NIH/MicroscopyForBeginnersReferenceGuide/blob/main/contributors.md#contributors) if that's a place where you would like to be acknowledged!

haesleinhuepf commented:
Hi @bethac07 ,

thanks for the feedback! I liked the suggestions and just implemented them. + I added some jupyter temp files to the gitignore.

Thanks again!

Best,
Robert
